lldb/source/Plugins/Language/Swift/SwiftOptional.cpp

```diff
@@ -116,12 +123,16 @@ ExtractSomeIfAny(ValueObject *optional,
     value_sp = value_sp->GetSyntheticValue();
 
   return value_sp;
-  }
+}
 
 static bool
 SwiftOptional_SummaryProvider_Impl(ValueObject &valobj, Stream &stream,
                                    const TypeSummaryOptions &options) {
-  PointerOrSP some = ExtractSomeIfAny(&valobj, true);
+  std::optional<ValueObjectSP> some_or_err = ExtractSomeIfAny(&valobj, true);
```

Comment:
is `_or_err` a naming convention for optionals too? I have only seen it for `Expected`.